app/models/social-link.js

```diff
@@ -32,3 +33,5 @@ export default ModelBase.extend({
 
   segmentedLink: computedSegmentedLink.bind(this)('link')
 });
+
+_.capitalize('normalizedName');
```

Comment:
Codacy found an issue: [Newline required at end of file but not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6232875)